content/en/docs/v3.6/metrics.md

```diff
@@ -110,6 +110,8 @@ The Prometheus client library provides a number of metrics under the `go` and `p
 | process_open_fds                  | Number of open file descriptors.           | Gauge        |
 | process_max_fds                   | Maximum number of open file descriptors.   | Gauge        |
 
+**Note**: The process metrics, such as process_open_fds and process_max_fds, are not supported on Darwin (macOS) systems at this time. This limitation should be considered when monitoring etcd instances running on Darwin systems.
```

Comment:
Suggest we keep it really simple and to the point.

```suggestion
**Note**: The process metrics, such as `process_open_fds` and `process_max_fds`, are not supported on Darwin (macOS) systems at this time.
```

Reply:
@jmhbnz Thanks for the feedback, I implemented it :)